sympy/vector/integrals.py

```diff
@@ -150,10 +151,31 @@ def vector_integrate(field, *region):
     >>> vector_integrate(C.x*C.i, region)
     12
 
+    Integrals over special regions can also be calculated using geometry module.
+    >>> from sympy.geometry import Point, Circle, Triangle
+    >>> c = Circle(Point(0, 2), 5)
+    >>> vector_integrate(C.x**2 + C.y**2, c)
+    290*pi
+    >>> triangle = Triangle(Point(-2, 3), Point(2, 3), Point(0, 5))
+    >>> vector_integrate(3*C.x**2*C.y*C.i + C.j, triangle)
+    -8
+
+    >>> vector_integrate(12*C.y**3, (C.y, 1, 3))
+    240
     >>> vector_integrate(C.x**2*C.z, C.x)
     C.x**3*C.z/3
+
     """
     if len(region) == 1:
         if isinstance(region[0], ParametricRegion):
             return ParametricIntegral(field, region[0])
+
+        if isinstance(region[0], GeometryEntity):
+            regions_list = parametric_region(region[0])
+
+            result = 0
```

Comment:
`result = S(0)` or `= S.Zero`... make sure it's a SymPy object and not a Python integer.

Reply:
ok.